source/fuzz/transformation_access_chain.cpp

```diff
@@ -433,4 +433,4 @@ std::unordered_set<uint32_t> TransformationAccessChain::GetFreshIds() const {
 }
 
 }  // namespace fuzz
-}  // namespace spvtools
+}  // namespace spvtools
```

Comment:
Please always include a newline at the end of a file.

Reply:
Done.